modules/services/easyeffects.nix

```
@@ -8,8 +8,59 @@ let
 
   presetOpts = optionalString (cfg.preset != "") "--load-preset ${cfg.preset}";
 
+  jsonFormat = pkgs.formats.json { };
+  presetType = types.submodule {
+    options = {
+      presetName = mkOption {
+        type = types.str;
+        description = "Name of the preset";
+      };
+      config = mkOption {
+        inherit (jsonFormat) type;
+        description =
+          "Content of the preset, see examples at https://github.com/wwmm/easyeffects/wiki/Community-Presets";
+      };
+    };
+  };
+  presetOptionType = mkOption {
+    type = types.listOf presetType;
+    default = [ ];
+    description = ''
+      List of presets to import to easyeffects.
+      Presets are written to `$XDG_CONFIG_HOME/easyeffects/{input,output}/{presetName}.json`.
+
+      See community presets at:
+      https://github.com/wwmm/easyeffects/wiki/Community-Presets
+    '';
+    example = literalExpression ''
+      [{
+        presetName = "my-preset";
+        config = {
+          input = {
+            blocklist = [
+
+            ];
+            "plugins_order" = [
+              "rnnoise#0"
+            ];
+            "rnnoise#0" = {
+              bypass = false;
+              "enable-vad" = false;
+              "input-gain" = 0.0;
+              "model-path" = "";
+              "output-gain" = 0.0;
+              release = 20.0;
+              "vad-thres" = 50.0;
+              wet = 0.0;
+            };
+          };
+        };
+      }]
```

Maybe this would be prettier as an attribute set, where the preset name is the attribute key. I.e., something like

```suggestion
      {
        my-preset = {
          input = {
            blocklist = [

            ];
            "plugins_order" = [
              "rnnoise#0"
            ];
            "rnnoise#0" = {
              bypass = false;
              "enable-vad" = false;
              "input-gain" = 0.0;
              "model-path" = "";
              "output-gain" = 0.0;
              release = 20.0;
              "vad-thres" = 50.0;
              wet = 0.0;
            };
          };
        };
      }
```